assets/src/edit-story/app/story/useStoryReducer/reducers/restore.js

```diff
@@ -12,12 +12,13 @@
  * @param {Object} payload New state to set.
  * @return {Object} New state
  */
-function restore( state, { pages, current, selection, story } ) {
+function restore( state, { pages, current, selection, story, capabilities } ) {
 	if ( ! Array.isArray( pages ) || pages.length === 0 ) {
 		return state;
 	}
 
 	const newStory = typeof story === 'object' ? story : {};
+	const newCapabilities = typeof story === 'object' ? capabilities : {};
```

Comment:
Should this be like this?

```suggestion
	const newCapabilities = typeof capabilities === 'object' ? capabilities : {};
```